tests/form.js

```diff
@@ -262,5 +264,116 @@ define([
             form.removeMugsFromForm(mugs);
             util.assertJSTreeState("");
         });
+
+        function assertInstanceSrc(id, form, expect, message) {
+            var xml = _.isString(form) ? form : form.createXML(),
+                $xml = $(xml),
+                result = $xml.find("instance[id='" + id + "']").attr("src");
```

Comment:
`head > model > ...` doesn't work (at least in phantomjs) because it doesn't like the namespaced `h:head`, but `model > ...` does.